extensions/amp-video/0.1/amp-video.js

```diff
@@ -622,13 +631,56 @@ class AmpVideo extends AMP.BaseElement {
     return ranges;
   }
 
+  /**
+    * Called when the video is first loaded.
+    * @override
+  */
+  firstLayoutCompleted() {
+    if (this.getPlaceholder() == null) {
+      this.togglePlaceholder(false);
+    }
```

Comment:
I'm reading this as "if there's no placeholder, hide it".

Reply:
Sorry forgot to push the commit that deleted this.